__tests__/index.test.js

```diff
@@ -99,4 +99,66 @@ describe('GET /api/articles/:article_id',()=>{
     });
 
 
-});
\ No newline at end of file
+})
+
+
+
+describe('GET /api/articles/:article_id/comments',()=>{
+    test('to get a 200 code',()=>{
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+    })
```

Comment:
^^ this test could be added to the test below :) You should be checking the code on each test you see :)